node/v2/handler.js

```diff
@@ -700,25 +686,15 @@ TChannelV2Handler.prototype.sendCallRequestContFrame = function sendCallRequestC
         self.errorEvent.emit(self, errors.SendCallReqContBeforeIdentifiedError());
         return;
     }
-    var reqBody = new v2.CallRequestCont(flags, req.checksum.type, args);
-    var result = self._sendCallBodies(req.id, reqBody, req.checksum);
-    req.checksum = result.checksum;
 
-    var req0 = self.connection.ops.getOutReq(req.id);
```

Comment:
req is a request continue frame that doesn't have serviceName etc. req0 is the first request sent out that has the info.